example/ddceler/compare_benchmarks.cc

```diff
@@ -0,0 +1,602 @@
+    legend_top->AddEntry(new TH1D(), "Statistical errors:", "f");
+    legend_top->AddEntry(h_config1_rel_err, "1#sigma", "f");
+    legend_top->AddEntry(h_config1_rel_err_3s, "3#sigma", "f");
+    legend_top->SetMargin(0.27);
+    legend_top->SetLineColor(kGray);
+    legend_top->Draw();
+
+    auto title_text = new TText(0.17, 0.92, hist_plot_title);
+    title_text->SetNDC();
+    title_text->SetTextColor(kGray);
+    title_text->Draw();
+
+    auto commit_text = new TLatex(0.67, 0.92, commit_hash);
+    commit_text->SetNDC();
+    commit_text->SetTextColor(kGray);
+    commit_text->Draw();
+
+    // Redraw axis above the histogram lines
+    pad_top->RedrawAxis();
+    pad_top->SetLogy();
+    // Move back to canvas
+    canvas->cd();
+
+    // Create bottom pad and move to it
+    auto pad_bottom = new TPad("pad2", "", 0.0, 0.0, 1.0, 0.3);
+    pad_bottom->SetTopMargin(0.02);
+    pad_bottom->SetBottomMargin(0.33);
+    pad_bottom->SetLeftMargin(0.11);
+    pad_bottom->Draw();
+    pad_bottom->cd();
+
+    h_config1_rel_err_3s->GetXaxis()->SetTitle(hist_x_axis);
+    h_config1_rel_err_3s->GetXaxis()->CenterTitle();
+    h_config1_rel_err_3s->GetXaxis()->SetTitleSize(0.14);
+    h_config1_rel_err_3s->GetXaxis()->SetTitleOffset(1.1);
+    h_config1_rel_err_3s->GetXaxis()->SetLabelSize(0.1153);
+    h_config1_rel_err_3s->GetXaxis()->SetLabelOffset(0.02);
+    h_config1_rel_err_3s->GetXaxis()->SetTickLength(0.07);
+
+    h_config1_rel_err_3s->GetYaxis()->SetTitle("Rel. Diff. (%)");
+    h_config1_rel_err_3s->GetYaxis()->CenterTitle();
+    h_config1_rel_err_3s->GetYaxis()->SetTitleSize(0.131);
+    h_config1_rel_err_3s->GetYaxis()->SetTitleOffset(0.415);
+    h_config1_rel_err_3s->GetYaxis()->SetLabelSize(0.116);
+    h_config1_rel_err_3s->GetYaxis()->SetLabelOffset(0.008);
+    h_config1_rel_err_3s->GetYaxis()->SetTickLength(0.04);
+    h_config1_rel_err_3s->GetYaxis()->SetNdivisions(503);
+    h_config1_rel_err_3s->GetYaxis()->SetRangeUser(-5, 5);
+
+    h_config1_rel_err_3s->SetLineColorAlpha(kGray, 0.7);
+    h_config1_rel_err_3s->SetFillColorAlpha(kGray, 0.7);
+    h_config1_rel_err_3s->SetMarkerSize(0);
+    h_config1_rel_err->SetLineColorAlpha(kGray + 1, 0.7);
+    h_config1_rel_err->SetFillColorAlpha(kGray + 1, 0.7);
+    h_config1_rel_err->SetMarkerSize(0);
+    h_config1_rel_err->GetYaxis()->SetRangeUser(-5, 5);
+    h_rel_diff->SetLineColor(config2_color);
+
+    // Draw stat. err. and rel. diff. histograms
+    h_config1_rel_err_3s->Draw("hist E2");
+    h_config1_rel_err->Draw("hist E2 sames");
+    h_rel_diff->Draw("hist sames");
+
+    pad_bottom->RedrawAxis();
+    canvas->SetLogy();
+    canvas->Print(config1_rootfile_label + "_" + config2_rootfile_label + "_"
+                  + plot_type + ".png");
+
+    // Clean up
+    delete canvas;
+    delete h_config1;
+    delete h_config2;
+    delete h_config1_rel_err;
+    delete h_config1_rel_err_3s;
+    delete h_rel_diff;
+}
```

Comment:
Exit with error if difference is too large. I.e. turn this into a test, not a plot.